gitclone.adoc

```diff
@@ -9,18 +9,15 @@
 ////
 == Getting started
 
-The fastest way to work through this guide is to clone its Git repository and use the provided projects
-inside. To do this, run the following commands:
+Run the following command to clone the Git repository and use the projects provided
+inside:
 
 [subs="attributes"]
 ----
 git clone https://github.com/openliberty/guide-{projectid}.git
 cd guide-{projectid}
 ----
 
-The `start` directory contains the starting project for this guide. Use this directory as the starting
-point for the guide.
+The `start` directory contains the starting project that you can use to begin.
```

Comment:
Perhaps we can say "The `start` directory contains the starting project that you can use to begin this guide."